client/src/views/page-components/account/history/history.ts

```diff
@@ -0,0 +1,99 @@
+import { $ } from '@src/utils/helper';
+import { createDOMWithSelector, matchCategoryAndImg } from '@src/utils/helper';
+import './history.scss';
+
+export default class History {
+  state: any;
+  history: HTMLElement;
+  constructor({ parent, state }) {
+    this.history = createDOMWithSelector('div', '.history');
+
+    parent.appendChild(this.history);
+    this.setProperty(state);
+    this.render();
+  }
+
+  setProperty(state): void {
```

Comment:
setProperty보단 setState가 어떨까요? property 라는 단어는 object에 보다 더 적합해보입니다
```suggestion
  setState(state): void {
```

Reply:
좋습니다 뭔가 리액트스러워서 탈출하고싶었지만 state로 정의한 이상 setState가 자연스럽네요!